_posts/2016-02-08-command-prompt-envy.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: post
+name: CommandPromptEnvy
+title: 'Command Prompt Envy'
+date: 2016-02-08 00:20:00 +00:00
+author: Pedro Santos
+image:
+    src: /assets/img/custom/blog/babun_screenshot.png
+tags:
+- command 
+- prompt
+- Babun
+- Consolas
+---
+
+I'm spending my time between Mac OS and Windows, and when I go back to my Windows machine I always feel the lack of my Mac command prompt. 
+
+Almost a year ago [Amir](https://twitter.com/amirbazazi) [posted](http://codurance.com/2015/03/16/installing-zprezto-a-quick-guide/) how you can get a nice command prompt on a Mac. I've always wanted a post like that, but for a Windows machine. Today I stumbled upon [Babun](http://babun.github.io/), gave it a try and TADA! I was able to get a command prompt exactly like the one I have on my Mac.
+
+###Follow these steps to get your shiny Windows command prompt:
+1. Download Babun from http://babun.github.io/ (272 MB) 
+ (Do not use Google Chrome to download since, for some reason, it will download the file but marks it as unsafe and deletes it).
+2. Unzip the downloaded archive
+3. Execute "install.bat" to install Babun to C:\Users\YOURUSER\.babun or execute "install.bat /t PATH_TO_INTALLATION_LOCATION"
```

Comment:
The github preview doesn't display the backslash between YOURUSER and .babun; can you confirm that this does appear on our site?